packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -137,7 +137,7 @@ class RealAccountSignup extends React.Component {
         if (has_real_account && currency) {
             if (is_eu && this.active_modal_index === modal_pages_indices.add_or_manage_account) {
                 // Manage account
-                return '379px'; // Since crypto is disabled for EU clients, lower the height of modal
+                return '420px'; // Since crypto is disabled for EU clients, lower the height of modal
```

Comment:
is it possible to use rem instead of px?

Reply:
No. We need to use pixels here.